__tests__/integration.test.js

```diff
@@ -192,3 +194,62 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("PATCH:201 - updates a given article by adding 1 vote", () => {
+    return request(app)
+      .patch("/api/articles/5")
+      .send({ inc_votes: 1 })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          body: "Bastet walks amongst us, and the cats are taking arms!",
+          created_at: "2020-08-03T13:14:00.000Z",
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          votes: 1,
+        });
+      });
+  });
+  test("PATCH:201 - updates a given article by adding multiple votes", () => {
+    return request(app)
+      .patch("/api/articles/5")
+      .send({ inc_votes: 100 })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          body: "Bastet walks amongst us, and the cats are taking arms!",
+          created_at: "2020-08-03T13:14:00.000Z",
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          votes: 100,
+        });
```

Comment:
The 201 tests are checking the same functionality, I'd recommend condensing it down into one. You could add another test to check that votes are decremented, for example: `.send({ inc_votes: -100 }`